atlasaction/action.go

```diff
@@ -869,21 +872,15 @@ func (a *Actions) SchemaPlanApprove(ctx context.Context) error {
 }
 
 // SchemaApply runs the GitHub Action for "ariga/atlas-action/schema/apply"
-func (a *Actions) SchemaApply(ctx context.Context) error {
-	params := &atlasexec.SchemaApplyParams{
-		ConfigURL:   a.GetInput("config"),
-		Env:         a.GetInput("env"),
-		Vars:        a.GetVarsInput("vars"),
-		DevURL:      a.GetInput("dev-url"),
-		URL:         a.GetInput("url"),
-		To:          a.GetInput("to"),
-		Schema:      a.GetArrayInput("schema"),
-		DryRun:      a.GetBoolInput("dry-run"),
-		AutoApprove: a.GetBoolInput("auto-approve"),
-		PlanURL:     a.GetInput("plan"),
-		TxMode:      a.GetInput("tx-mode"), // Hidden param.
+func (a *Actions) SchemaApply(ctx context.Context) (err error) {
+	var results []*atlasexec.SchemaApply
+	// Determine if the approval process should be used.
+	useApproval := a.GetInput("auto-generate-plan") != "" && !a.GetBoolInput("auto-approve")
```

Comment:
```suggestion
	useApproval := a.GetInput("approval_policy") != "" && !a.GetBoolInput("auto-approve")
```
I think we should stick to the same terms/names we use in k8s. 
cc @rotemtam, @giautm